src/pages/subpage/Subreddit.js

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import {useHistory, useParams} from "react-router-dom";
+import {useEffect, useState } from "react";
+import axios from "axios";
+import './Subreddit.css';
+import svgImage from '../assets/loadingIcon.gif'
+
+function Subreddit ({topRedditPosts}) {
+    const [subData, setSubData] = useState('');

```

Comment:
Het zal per definitie nooit een string zijn, maar een object. Daarom zou ik als default waarde `undefined` of `{}` nemen